app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -53,7 +53,7 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
-        if (datum != null) {
```

Comment:
Don't remove datum != null . Keep it there. Add null check for title like
```
if( datum.getDescription == null || datum.getDescription().isEmpty()) {
holder.descriptionTextView.setVisibility(View.GONE);
} else {
holder.descriptionTextView.setVisibility(View.VISIBLE);
holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
}
```
replace description with title. If not null and not empty then only display title else hide it.